src/Persistence/MassTransit.MongoDbIntegration/MongoDbIntegration/Outbox/BusOutboxNotification.cs

```diff
@@ -0,0 +1,49 @@
+namespace MassTransit.MongoDbIntegration.Outbox
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Extensions.Options;
+
+
+    public class BusOutboxNotification :
```

Comment:
Seems like this could be in the main MassTransit assembly, nothing unique to MongoDB or Entity Framework?

Reply:
it was like that before you commented :D

Reply:
Assembly, yes. `MassTransit` namespace: no. :)